DakeAndDevileCorps/Screens/MapHome/Component/SearchBarView.swift

```diff
@@ -114,23 +146,26 @@ class SearchBarView: UIView {
             containerView.topAnchor.constraint(equalTo: topAnchor),
             containerView.bottomAnchor.constraint(equalTo: bottomAnchor),
             containerView.leadingAnchor.constraint(equalTo: leadingAnchor),
-            containerView.trailingAnchor.constraint(equalTo: trailingAnchor)
+            containerView.trailingAnchor.constraint(equalTo: trailingAnchor),
         ])
         
+        
         containerView.addSubview(textField)
+        let textFieldTrailingConstraintContainerView =  textField.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -8)
```

Comment:
p1;
= 과 textField 사이에 두 칸 띄우신 것 같습니다. 수정 부탁드려용!🥸
```suggestion
        let textFieldTrailingConstraintContainerView = textField.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -8)
```

Reply:
넵 수정했습니당
[9ae7455](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/40/commits/9ae74558c022ffad29a2cef2103c7efc173eb8be)